src/views/Community/Header.tsx

```diff
@@ -45,7 +45,7 @@ const Header = ({ community, updateReview }: any) => {
                     <Box flex>
                         {(community?.review === 'pending' ||
                             community?.review === 'declined') && (
-                            <Box>
+                            <Box fGrow={1} fLayout={{ sm: "end", xs: "start"}} flex>
```

Comment:
Strangely, this should fire a warning through linter - use single quotes instead double quotes (")